protocol/contracts/libraries/LibStrings.sol

```diff
@@ -51,4 +51,15 @@ library LibStrings {
     function toHexString(address addr) internal pure returns (string memory) {
         return toHexString(uint256(uint160(addr)), _ADDRESS_LENGTH);
     }
+
+    /**
+     * @dev Converts a `int256` to its ASCII `string` representation.
+     */
+    function toString(int256 value) internal pure returns(string memory){
+        if(value > 0){
```

Comment:
Is there a reason you don't put spaces before and after the parentheses in your if statements?

Reply:
mistake, fixed to match style guide: c58be9b7fa3eda06805602e40e1e93f0fb281137